numba/unicode.py

```diff
@@ -199,6 +198,19 @@ def details(context, builder, signature, args):
     return sig, details
 
 
+@intrinsic
```

Comment:
Perhaps this ought to have `align` added as an argument to make it more general and then it could go into `unsafe` as a utility function?

Reply:
good idea.  Moved function to `memcpy_region` and added a unit test for it.